pipeline-steps/templates/steps/vault-upload.yaml

```diff
@@ -15,12 +15,12 @@ steps:
             varValue='$(${{ pair.env_var_name }})'
```

Comment:
change the name to var_value as it does not represent environment var name anymore